From 768d9427b1a8e5b3308cb518984cd35e950bd6c5 Mon Sep 17 00:00:00 2001
From: Ma Wupeng <mawupeng1@huawei.com>
Date: Mon, 25 Jul 2022 10:23:07 +0800
Subject: [PATCH 3526/4585] [Huawei] mm: Drop shmem reliable related log during
 startup

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187270

--------------------------------

Message "shmem reliable disabled." will be printed if memory reliable is
disabled. This is not necessary so drop it.

Signed-off-by: Ma Wupeng <mawupeng1@huawei.com>
---
 mm/mem_reliable.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/mm/mem_reliable.c b/mm/mem_reliable.c
index b1bc749532a4..15a84c0f714e 100644
--- a/mm/mem_reliable.c
+++ b/mm/mem_reliable.c
@@ -101,13 +101,8 @@ void mem_reliable_init(bool has_unmirrored_mem, unsigned long *zone_movable_pfn)
 
 void shmem_reliable_init(void)
 {
-	if (!shmem_reliable_is_enabled())
-		return;
-
-	if (!mem_reliable_is_enabled()) {
+	if (!mem_reliable_is_enabled() || !shmem_reliable_is_enabled())
 		shmem_reliable = false;
-		pr_info("shmem reliable disabled.\n");
-	}
 }
 
 void reliable_report_meminfo(struct seq_file *m)
-- 
2.17.1

